src/App.css

```diff
@@ -1,3 +1,57 @@
+* {
+  margin: 0px;
+  padding: 0px;
+}
+
 .App {
   text-align: center;
 }
+
+.images {
+  width: 70%;
```

Comment:
This needs a bit of adjustment for desktop view - at the moment, the images are massive on desktop.

Reply:
try img {
max-width: 100%
}

Near the top of the css styling page